Makefile

```diff
@@ -180,6 +180,10 @@ else
 endif
 export KUSTOMIZE
 
+install: operator-sdk
```

Comment:
May be we can call the target install-all ?

Reply:
once we remove other targets in the coming PR we will have only one target which is called install.